networkx/classes/function.py

```diff
@@ -156,7 +156,10 @@ def degree_histogram(G):
     (Order(number_of_edges))
     """
     counts = Counter(d for n, d in G.degree())
-    return [counts.get(i, 0) for i in range(max(counts) + 1)]
+    if len(counts) > 0:
```

Comment:
I know Otis the same thing, but can we please use the “inline if” here?
```python
    return [counts.get(i, 0) for i in range(max(counts) + 1 if counts else 0)]
```
I believe `if counts` is the preferred style over `if len(counts) >0` because we know counts is a Counter.

Reply:
Understood, I'll make this change and commit